core/nwb.base.yaml

```
@@ -249,6 +249,14 @@ groups:
       stored external to the NWB file, in files storing raw data. Once timestamp
       data is calculated, the contents of 'sync' are mostly for archival purposes.
     quantity: '?'
+  links:
+  - name: device
```

```suggestion
  - name: acquisition_device
```

Maybe this would make the expected use more clear?